src/Admin/VolunteerAdmin.php

```diff
@@ -0,0 +1,139 @@
+    {
+        $listMapper
+            ->add('lastName', null, [
+                'label' => 'Nom',
+            ])
+            ->add('firstName', null, [
+                'label' => 'Prénom',
+            ])
+            ->add('emailAddress', null, [
+                'label' => 'E-mail',
+            ])
+            ->add('favoriteCities', null, [
+                'label' => 'Ville(s) choisie(s)',
+            ])
+            ->add('isAdherent', 'boolean', [
+                'label' => 'Adherent',
+            ])
+            ->add('_action', null, [
+                'actions' => [
+                    'edit' => [],
+                    'delete' => [],
+                ],
+            ])
+        ;
+    }
+
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        $formMapper
+            ->add('lastName', null, [
+                'label' => 'Nom',
+            ])
+            ->add('firstName', null, [
+                'label' => 'Prénom',
+            ])
+//            ->add('favoriteCities', null, [
+//                'label' => 'Ville(s) choisie(s)',
+//            ])
+            ->add('emailAddress', null, [
+                'label' => 'E-mail',
+            ])
+            ->add('address', null, [
+                'label' => 'Adresse',
+            ])
+            ->add('postalCode', null, [
+                'label' => 'Code postale',
+            ])
+            ->add('city', null, [
+                'label' => 'Ville',
+            ])
+            ->add('country', CountryType::class, [
+                'label' => 'Pays',
+            ])
+            ->add('phone', PhoneNumberType::class, [
+                'label' => 'Téléphone',
+                'widget' => PhoneNumberType::WIDGET_COUNTRY_CHOICE,
+            ])
+            ->add('profession', null, [
+                'label' => 'Profession',
+            ])
+            ->add('favoriteThemes', EntityType::class, [
+                'label' => 'Thèmes favoris',
+                'class' => Theme::class,
+                'multiple' => true,
+            ])
+            ->add('customFavoriteTheme', null, [
+                'label' => 'Thèmes favoris personnalisé',
+            ])
+            ->add('technicalSkills', EntityType::class, [
+                'label' => 'Compétences techniques',
+                'class' => TechnicalSkill::class,
+                'multiple' => true,
+            ])
+            ->add('customTechnicalSkills', null, [
+                'label' => 'Compétences techniques personnalisées',
+            ])
+            ->add('isPreviousCampaignMember', BooleanType::class, [
+                'label' => 'Fait partie de d\'une précédente campagne ?',
+            ])
+            ->add('previousCampaignDetails', null, [
+                'label' => 'Détails de la précédente campagne',
+            ])
+            ->add('shareAssociativeCommitment', BooleanType::class, [
+                'label' => 'Partage l\'engagement associatif ?',
```

Comment:
Can you use this syntax with `".."` to avoid backslashes please ? 
=> `"Partage l'engagement associatif ?"`

Same for others.